--- origsrc/qttools-opensource-src-5.1.0/src/designer/src/uitools/uitools.pro	2013-07-02 02:10:51.000000000 -0500
+++ src/qttools-opensource-src-5.1.0/src/designer/src/uitools/uitools.pro	2013-08-15 13:19:07.416472600 -0500
@@ -1,5 +1,4 @@
 TARGET = QtUiTools
-CONFIG += static
 QT = core
 
 QMAKE_DOCS = $$PWD/doc/qtuitools.qdocconf
@@ -10,7 +9,6 @@ HEADERS += quiloader.h
 SOURCES += quiloader.cpp
 
 DEFINES += \
-    QFORMINTERNAL_NAMESPACE \
-    QT_DESIGNER_STATIC
+    QFORMINTERNAL_NAMESPACE
 
 include(../lib/uilib/uilib.pri)
